client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue

```diff
@@ -355,7 +396,11 @@ export default {
     },
 
     hasActions () {
-      return this.actions.addRecommendations.checked || this.actions.addTags.selected.length > 0 || this.actions.deleteTags.selected.length > 0 || Object.values(this.actions.assignSegment.selected).length > 0
+      return this.actions.addRecommendations.checked || this.actions.addTags.selected.length > 0 || this.actions.deleteTags.selected.length > 0 || Object.values(this.actions.assignSegment.selected).length > 0 || Object.values(this.actions.assignPlace.selected).length > 0
```

Comment:
The expression is a little hard to read... maybe we could extract some logic into separate `const` blocks. Also when looking into the logic here, i spotted 2 bugs (that were already present before this change but could be fixed along the way):

- Checking any of the "Zur Bearbeitung zuweisen", "Schlagworte hinzufügen..." or "Schlagworte entfernen..." boxes, selecting something, and unchecking the box again does not set `hasAction` to false, as the expression does not test for `this.actions.<action>.checked` at this point.
- Checking the "Erwiderung verfassen..." box sets `hasAction` to true even if no text is given, because the expression does not test for `this.actions.addRecommendations.text` 

As each box has to be treated individually here, i'd suggest to structure it like so:

```js
const addRecommendationAction = this.actions.addRecommendations.checked && this.actions.addRecommendations.text
// ...
const deleteTagsAction = this.actions.deleteTags.checked && this.actions.deleteTags.selected.length > 0

return addRecommendationAction || ... || deleteTagsAction
```

Reply:
it is a smart idea, thanks! done here https://github.com/demos-europe/demosplan-core/pull/695/commits/b885ee7f6aee76ee922933980cd21cbce309f924